PlaywrightDemoQA/WebTablesTest.cs

```diff
@@ -0,0 +1,86 @@
+﻿using Microsoft.Playwright;
+
+namespace PlaywrightTests;
+
+[Parallelizable(ParallelScope.Self)]
+[TestFixture]
+public class WebTablesTest
+{
+    [Test]
+    public async Task DeleteRow()
+    {
+        using var playwright = await Playwright.CreateAsync();
+        await using var browser = await playwright.Chromium.LaunchAsync(new BrowserTypeLaunchOptions
+        {
+            Headless = false,
+        });
+        var context = await browser.NewContextAsync();
+
+        var page = await context.NewPageAsync();
+        await page.GotoAsync("https://demoqa.com/webtables");
+        await page.Locator(".rt-tr-group")
+            .Filter(new() { Has = page.GetByRole(AriaRole.Gridcell, new() { Name = "Alden", Exact = true }) })
+            .GetByTitle("Delete").ClickAsync();
+        await Assertions.Expect(page.GetByText("Alden")).Not.ToBeVisibleAsync();
+    }
+
+    [Test]
+    public async Task EditLastNameandSallary()
+    {
+        using var playwright = await Playwright.CreateAsync();
+        await using var browser = await playwright.Chromium.LaunchAsync(new BrowserTypeLaunchOptions
+        {
+            Headless = false,
+        });
+        var context = await browser.NewContextAsync();
+
+        var page = await context.NewPageAsync();
+        await page.GotoAsync("https://demoqa.com/webtables");
+        await page.Locator(".row > div:nth-child(3)").ClickAsync();
+          await page.Locator("#edit-record-2 path").ClickAsync();
+        await page.GetByPlaceholder("Last Name").ClickAsync();
+        await page.GetByPlaceholder("Last Name").FillAsync("Cantrell edit");
+        await page.GetByPlaceholder("Salary").FillAsync("15000");
+        await page.GetByRole(AriaRole.Button, new() { Name = "Submit" }).ClickAsync();
+        await Assertions.Expect(page.Locator(".col-12 > div:nth-child(4)")).ToBeVisibleAsync();
+
+
+        await Assertions.Expect(page.GetByRole(AriaRole.Grid)).ToContainTextAsync("Cantrell edit");
```

Comment:
this assert verifies that "Cantrell edit" is present somewhere in the table. This is better than nothing, but it should at least verify that this text is present in Alden's row. Ideally it should verify that Alden's last name is "Cantrell edit"